libr/bin/format/elf/elf.c

```diff
@@ -1821,6 +1822,7 @@ ut64 Elf_(r_bin_elf_get_main_offset)(ELFOBJ *bin) {
 		return UT64_MAX;
 	}
 	// unnecessary to read 512 bytes imho
+	ut8 buf[256];
```

Comment:
what is this shadowing here? seems bad practice

Reply:
its a bad rebasing, the same issue happened with the size_t in 1730. im fixing them :) that was a blind rebase of another PR of me. thanks for the review!